lib/stats/tests/test_stats_cluster.c

```diff
@@ -231,42 +222,22 @@ test_get_component_name_translates_component_to_name_properly(void)
   assert_stats_component_name(SCS_DESTINATION | SCS_GROUP, "destination");
 }
 
-static void
-test_get_counter(void)
+Test(stats_cluster, test_get_counter)
 {
   StatsClusterKey sc_key;
   stats_cluster_logpipe_key_set(&sc_key, SCS_SOURCE | SCS_FILE, "id", "instance" );
   StatsCluster *sc = stats_cluster_new(&sc_key);
   StatsCounterItem *processed;
 
-  assert_true(stats_cluster_get_counter(sc, SC_TYPE_PROCESSED) == NULL, "get counter before tracked");
+  cr_assert_null(stats_cluster_get_counter(sc, SC_TYPE_PROCESSED), "get counter before tracked");
   processed = stats_cluster_track_counter(sc, SC_TYPE_PROCESSED);
-  assert_true(stats_cluster_get_counter(sc, SC_TYPE_PROCESSED) == processed, "get counter after tracked");
+  cr_assert_eq(stats_cluster_get_counter(sc, SC_TYPE_PROCESSED), processed, "get counter after tracked");
 
   StatsCounterItem *saved_processed = processed;
   stats_cluster_untrack_counter(sc, SC_TYPE_PROCESSED, &processed);
-  assert_true(processed == NULL, "untrack counter");
-  assert_true(stats_cluster_get_counter(sc, SC_TYPE_PROCESSED) == saved_processed, "get counter after untracked");
+  cr_assert_null(processed, "untrack counter");
+  cr_assert_eq(stats_cluster_get_counter(sc, SC_TYPE_PROCESSED), saved_processed, "get counter after untracked");
   stats_cluster_free(sc);
 }
 
-static void
-test_stats_cluster(void)
-{
-  STATS_CLUSTER_TESTCASE(test_stats_cluster_new_replaces_NULL_with_an_empty_string);
-  STATS_CLUSTER_TESTCASE(test_stats_cluster_equal_if_component_id_and_instance_are_the_same);
-  STATS_CLUSTER_TESTCASE(test_stats_foreach_counter_yields_tracked_counters);
-  STATS_CLUSTER_TESTCASE(test_stats_foreach_counter_never_forgets_untracked_counters);
-  STATS_CLUSTER_TESTCASE(test_get_component_name_translates_component_to_name_properly);
-  STATS_CLUSTER_TESTCASE(test_stats_cluster_single);
-  STATS_CLUSTER_TESTCASE(test_stats_cluster_key_not_equal_when_custom_tags_are_different);
-  STATS_CLUSTER_TESTCASE(test_stats_cluster_key_equal_when_custom_tags_are_the_same);
-  STATS_CLUSTER_TESTCASE(test_get_counter);
-}
-
-int
-main(int argc, char *argv[])
-{
-  test_stats_cluster();
-  return 0;
-}
+TestSuite(stats_cluster);
```

Comment:
Btw `TestSuit` is not required to write for Criterion.

Reply:
okay. I will keep that in mind.